data/xml/D18.xml

```diff
@@ -25,6 +25,7 @@
       <url>D18-1001</url>
       <abstract>This article deals with adversarial attacks towards deep learning systems for Natural Language Processing (NLP), in the context of privacy protection. We study a specific type of attack: an attacker eavesdrops on the hidden representations of a neural text classifier and tries to recover information about the input text. Such scenario may arise in situations when the computation of a neural network is shared across multiple devices, e.g. some hidden representation is computed by a user’s device and sent to a cloud-based model. We measure the privacy of a hidden representation by the ability of an attacker to predict accurately specific private information from it and characterize the tradeoff between the privacy and the utility of neural representations. Finally, we propose several defense methods based on modified training objectives and show that they improve the privacy of neural representations.</abstract>
       <video href="https://vimeo.com/305202770" tag="video"/>
+      <doi>10.18653/v1/D18-0101</doi>
     </paper>
```

Comment:
This one isn't correct—it should be D18-1001. I wonder if you used the old buggy `construct_anthology_url`?